contributing/publishing.md

```diff
@@ -28,19 +25,6 @@ For more information about publishing take a look at:
 
 The release branch is typically created from a scheduled job in CircleCI. This scheduled job creates the release branch off of the `develop` branch on Tuesdays at 7 PM PST. Release branches are in the format `release/vxx.yy.zz`.
 
-To create a release branch manually:
```

Comment:
Thanks for moving this section below to the manual section. It is confusing having this here 👍 .